app/serializers/preprint.ts

```diff
@@ -1,6 +1,19 @@
+import Model from '@ember-data/model';
+import { Resource } from 'osf-api';
+
 import OsfSerializer from './osf-serializer';
 
 export default class PreprintSerializer extends OsfSerializer {
+    normalize(modelClass: Model, resourceHash: Resource) {
+        const result = super.normalize(modelClass, resourceHash);
+        // Insert a `versions` relationship to the model
```

Comment:
I did not realize we weren't serializing this on the back end. Ick. Not blocking, but I'm not super-happy about it.